cypress/e2e/pages/vaults/[vaultid]/index.spec.ts

```diff
@@ -269,13 +278,15 @@ context("/vaults/[vaultid] on mobile", () => {
     });
 
     it("should have loan details card", () => {
-      cy.findByTestId("VaultCollapsibleSection.LoanDetails").within(() => {
-        cy.findAllByTestId("LoanDetailsCard").within(() => {
-          cy.findByTestId("LoanDetailsCard.AssetIcon").should("be.visible");
-          cy.findByTestId("LoanDetailsCard.displaySymbol").should("be.visible");
-          cy.findByTestId("LoanDetailsCard.LoanValue").should("be.visible");
-          cy.findByTestId("LoanDetailsCard.LoanAmount").should("be.visible");
-          cy.findByTestId("LoanDetailsCard.TotalInterestRate").should(
+      cy.findAllByTestId("VaultCollapsibleSection.LoanDetails").each(($el) => {
+        cy.wrap($el).within(() => {
+          cy.findAllByTestId("LoanDetailsCard.AssetIcon").should("be.visible");
+          cy.findAllByTestId("LoanDetailsCard.displaySymbol").should(
+            "be.visible"
```

Comment:
Can we make this into single line like this `cy.findByTestId("CollateralCard.AssetIcon").should("be.visible");` to maintain its readability?